hot_osm/templates/components/branded_elements/button.html

```diff
@@ -0,0 +1,3 @@
+<button class="font-medium font-barlow h-hot-4 text-base-18 bg-hot-red text-hot-white px-hot-3 rounded-8 border-2pt border-transparent hover:bg-hot-off-white hover:text-hot-red focus:bg-transparent focus:text-hot-red focus:outline-none nav-breakpoint:ml-3 {{ classes }}">
```

Comment:
Why do we have a transparent border here? - also related to the comment on using pt :)